File: WEEK1/DAY4/Prog3/P3.c
#include <stdio.h>
int main(){
    printf("Check whether a character is vowel or consonant\n");
    char c;
    printf("Enter character: ");
    scanf("%c",&c);
    if ( (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') ) {
        if(c=='a'||c=='A'||c=='e'||c=='E'||c=='i'||c=='I'||c=='o'||c=='O'||c=='u'||c=='U'){
            printf("Vowel\n");
        }
        else{
            printf("Consonant\n");   
        }
    }
    else{
        printf("This is not an alphabet\n");
    }
}

File: WEEK1/DAY4/Prog3/P3.py
print("Check whether a character is vowel or consonant")
vowel="aeiou"
char=input("Enter character: ")
if 'a'<=char<='z' or 'A'<=char<='Z':
    if char in vowel:
        print("Vowel!")
    else:
        print("Consonant")
else:
    print("This is not an alphabet")
